package/tests/test_api/test_apps.py

```diff
@@ -284,23 +284,23 @@ def assert_example_transcoded_data(response_data):
     # compare nodes
     expected_nodes = [
         {
-            "id": "f2b74919",
+            "id": "2a1abe98",
             "name": "Process Data",
             "full_name": "process_data",
             "tags": ["split"],
-            "pipelines": ["__default__", "data_processing"],
-            "modular_pipelines": [],
```

Comment:
just out of curiosity, I wonder the need behind deleting the `modular_pipelines` field in the test? ( this doesn't seem to relate to the task on hand, hence my curiousity)

Reply:
so windows builds were failing on 3.9, 3.10 due to some python dependency issues. this was because in kedro-viz we import multiple kedro datasets i.e. pandas, and spark which we use for python tests. 
in order for us to have less python dependencies,i removed the spark datasets in the python tests and replaced it with pandas dataset. Hence i had to edit the python tests accordingly.